### PR TITLE
[hist] TGraph/TH1::SaveAs title was using incoherent delimiter

### PR DESCRIPTION
**Pull request by ferdymercury**

### CODE REVIEW

File: hist/hist/src/TH1.cxx [hunk 1]
```diff
@@ -7179,11 +7179,11 @@ void TH1::SaveAs(const char *filename, Option_t *option) const
    }
    if (opt.Contains("title")) {
       if (std::strcmp(GetYaxis()->GetTitle(), "") == 0) {
-         out << "#\tBinLowEdge\tBinUpEdge\t"
+         out << "# " << "BinLowEdge" << del << "BinUpEdge" << del
              << "BinContent"
-             << "\tey" << std::endl;
+             del << "ey" << std::endl;
```
Comment (ferdymercury) [inline hist/hist/src/TH1.cxx:7184]:
```suggestion
             << del << "ey" << std::endl;
```

**Review by couet — APPROVED** [review]
LGTM

**Review by jblomer — COMMENTED** [review]
OK for the RField fix (but looks unrelated to the rest of the PR).

**Comment by ferdymercury** [comment]
@dpiparo I believe this one is ready to merge, thks!

**Comment by couet** [comment]
@ferdymercury as @jblomer pointed it seems the RField fix is not related to the rest of the PR. Do we want to merge it anyway?

**Comment by ferdymercury** [comment]
Given it's in a separate commit and it's a skip-ci commit, I'd say it's ok as long as squash it's not applied

**Pull request merged**